fuzz/fuzz_targets/iommu.rs

```diff
@@ -66,6 +66,7 @@ fuzz_target!(|bytes: &[u8]| -> Corpus {
         SeccompAction::Allow,
         EventFd::new(EFD_NONBLOCK).unwrap(),
         ((MEM_SIZE - IOVA_SPACE_SIZE) as u64, (MEM_SIZE - 1) as u64),
+        64,
```

Comment:
const MAX_IOMMU_ADDRESS_WIDTH_BITS: u8 = 64;

Reply:
Since you've asked me to split the comments above, and @rbradford stated that commits should be independently compilable (atomic), this would not be possible to achieve because:

* `virtio-devices: iommu: allow limiting maximum address width in bits` contains this line
* `vmm: introduce platform option to limit maximum IOMMU address width` contains the `MAX_IOMMU_ADDRESS_WIDTH_BITS` constant

🤷